tests/playwright/src/utility/cleanup.ts

```diff
@@ -25,8 +25,8 @@ import { existsSync, rmSync } from 'node:fs';
 export async function removeFolderIfExists(path: string): Promise<void> {
   console.log(`Cleaning up folder: ${path}`);
 
-  if (existsSync(path)) {
+  if (fs.existsSync(path)) {
     console.log(`Folder found, removing...`);
-    rmSync(path, { recursive: true, force: true, maxRetries: 5 });
+    fs.rmSync(path, { recursive: true, force: true, maxRetries: 5 });
```

Comment:
not sure it's better to import using the wildcard there ??

Reply:
I have added specific imports.

Reply:
thanks